test/psych/test_exception.rb

```diff
@@ -94,7 +94,7 @@ def test_psych_parse_takes_file
       assert_nil ex.file
 
       ex = assert_raises(Psych::SyntaxError) do
-        Psych.parse '--- `', 'omg!'
+        Psych.parse '--- `', filename: 'omg!'
```

Comment:
I wonder if it makes sense to add a new test that just uses the deprecated API to make sure it's deprecated but still useable.

cc @tenderlove